metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -558,6 +558,10 @@ def extract_dbt_entities(
         if owner is None:
             owner = manifest_node.get("config", {}).get("meta", {}).get("owner")
 
+        schema = manifest_node.get("config", {}).get("schema")        
+        if schema is None:
+            schema = manifest_node.get("schema")
```

Comment:
I think this variable needs to get used below

Also, does this only apply to schema or should we make the change to `database` too?